### PR TITLE
Add 404 page and redirect path

### PR DESCRIPTION
**Pull request by claudiusbr**
Relates to prometheus#1795

still TODO: 
- add link to latest docs. 

This is what the page looks like when I run it:

image

As this is my first contribution, I thought it would be a good idea to start the discussion by opening a PR. Hope this is ok, and look forward to some suggestions on how to improve.

### CODE REVIEW

File: content/_redirects [hunk 1]
```diff
@@ -39,6 +39,7 @@
 /docs/alerting/             /docs/alerting/latest/overview/             302!
 /docs/practices/            /docs/practices/naming/                     302!
 /docs/guides/               /docs/guides/basic-auth/                    302!
+/docs/*                     /docs/404/                                  302!
```
Comment (claudiusbr) [inline content/_redirects:42]: I had a read through, and seems to me this is the way to do a redirect. But have not been able to test it on my local, so I'd be grateful for pointers on how to do it.

**Comment by juliusv** [comment]
Yep that works, thanks :) I accepted the deploy preview on Netlify, which you can try here now: https://deploy-preview-2400--prometheus-docs.netlify.app/docs/404/

However it would be nice to have this for anything on prometheus.io, not just under /docs - so maybe we can just change the `/docs/*` to `/*`?

**Comment by claudiusbr** [comment]
Thanks for the review @juliusv! 
I changed it to  `/*` now. Hope it still works.

**Comment by juliusv** [comment]
I retriggered the preview deploy and it's now also redirecting for pages that should exist. I think I found out why: the Netlify documentation has this example: https://docs.netlify.com/routing/redirects/redirect-options/#custom-404-page-handling and this explanation about shadowing and overrides: https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing

I see two issues if I understand this correctly:

* The rule shouldn't return a 302 redirect code, but return a 404 (as in the Netlify example)
* We want to remove the exclamation mark at the end to not return the 404 page for pages that exist

**Comment by claudiusbr** [comment]
Great. It makes semantic sense too.
I'm not that familiar with Netlify (yet), but hope it works now.
I'm assuming I can't re-trigger the preview deploys myself, so thanks for doing it @juliusv.

**Comment by juliusv** [comment]
Noice! Looks good now, thanks :)

**Pull request merged**